### DISCUSSION TITLE
Hide installed packages without version?

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco-CMS#14644] [Features and ideas]
It seems Umbraco pick up folders from App_Plugins with package.manifest (but not folders without one) and packages using `PackageManifestFilter` like https://github.com/dawoe/OEmbed-Picker-Property-Editor/issues/31

However often this are used to inject custom CSS or JS.

![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/2f6c6796-919c-4ecd-9b38-aecd09386daa)

![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/48c30b30-1149-4b70-89ac-db70802a5c80)

Should it only pickup packages registered with a version? or maybe version, alias or name?

So something like this isn't shown as a package?

```
{
    "css": [
        "~/App_Plugins/MyPlugin/css/styles.css"
    ],
    "javascript": [
        "~/App_Plugins/MyPlugin/js/scripts.js"
    ]
}
```

**Comment by bergmania** [comment]
Hi @bjarnef.

What's the problem with those extensions/packages are listed? 

Personally I prefer all extensions to be listed

**Reply by bjarnef** [reply]
Not a big deal, but possible a project can contains a lot of package.manifest files to inject CSS or JS .. it may be considered as extensions, but not sure if packages are the right word here.
It may clutter with actual packages installed.

Also the "MCB.Umbraco.RTE" isn't listed, because it doesn't contain any package.manifest, just a few static HTML templates to use template plugin in richtext editor.

Maybe the items could be grouped?

**Installed packages**
- Contains either version, alias or package name (e.g. the OEmbed property editor doesn't set alias or version, but package name).

**Extensions** 
- Any other other registered extensions with doesn't match the first group.

**Reply by bergmania** [reply]
i don't have strong opinions neither for or agains, so if a PR is made, it's just important another is made for v14. Alternatively only do it for v14
